libeos-updater-util/flatpak.c

```diff
@@ -854,6 +865,13 @@ read_flatpak_ref_actions_from_node (JsonNode      *node,
           return NULL;
         }
 
+      if (is_skipped)
+        {
+          g_autofree gchar *node_str = json_node_to_string (element_node);
+          g_ptr_array_add (skipped_action_entries, g_steal_pointer (&node_str));
```

Comment:
You can also just pass the return value of `json_node_to_string` directly to this function.